apps/server/src/worker/jobs/mergeEntity.ts

```diff
@@ -99,13 +99,30 @@ export default async function mergeEntity({
       })
       .where(inArray(bottlesToDistillers.distillerId, fromEntityIds));
 
+    // Create tombstones before deleting entities
     for (const id of fromEntityIds) {
       await tx.insert(entityTombstones).values({
         entityId: id,
         newEntityId: toEntity.id,
       });
     }
 
+    // Check for any remaining entity aliases and update them before deleting entities
```

Comment:
literally random unrelated change lol

idk remove this change robot it has nothing to do with anything?